OpenMTRDemo/Forms/TiledFiltersForm.cs

```diff
@@ -0,0 +1,391 @@
+
+            Cv2.CvtColor(grayMat, grayMat, ColorConversionCodes.BGR2GRAY);
+            Cv2.Canny(grayMat, cannyMat, 225, 255);
+            Cv2.Sobel(sobelMat, sobelMat, MatType.CV_8U, xorder: 1, yorder: 0, ksize: -1);
+            Cv2.Laplacian(laplacianMat, laplacianMat, MatType.CV_8U, 3, 1, 0);
+            Cv2.Scharr(grayMat, scharrMat, MatType.CV_8U, xorder: 0, yorder: 1);
+
+            DemoUtilities.loadImage(pictureBoxSource, meter.SourceImage);
+            DemoUtilities.loadImage(pictureBoxPane1, meterPane1.SourceImage);
+            DemoUtilities.loadImage(pictureBoxPane2, meterPane2.SourceImage);
+            DemoUtilities.loadImage(pictureBoxPane3, meterPane3.SourceImage);
+            DemoUtilities.loadImage(pictureBoxPane4, meterPane4.SourceImage);
+            DemoUtilities.loadImage(pictureBoxPane5, meterPane5.SourceImage);
+            DemoUtilities.loadImage(pictureBoxPane6, meterPane6.SourceImage);
+            DemoUtilities.loadImage(pictureBoxGray, grayMat);
+            DemoUtilities.loadImage(pictureBoxSobel, sobelMat);
+            DemoUtilities.loadImage(pictureBoxCanny, cannyMat);
+            DemoUtilities.loadImage(pictureBoxLaplace, laplacianMat);
+            DemoUtilities.loadImage(pictureBoxScharr, scharrMat);
+        }
+
+        private void pictureBoxSource_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(meter.SourceImage);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxGray_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(grayMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxCanny_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(cannyMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxSobel_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(sobelMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxLaplace_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(laplacianMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxScharr_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(scharrMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxPane1_DoubleClick(object sender, EventArgs e)
```

Comment:
There are 6 of these kinds of handlers. Is there anyway to consolidate them to a single handler?